airflow/api_connexion/endpoints/connection_endpoint.py

```diff
@@ -129,3 +130,24 @@ def post_connection(session):
         session.commit()
         return connection_schema.dump(connection)
     raise AlreadyExists(detail=f"Connection already exist. ID: {conn_id}")
+
+
+@security.requires_access([(permissions.ACTION_CAN_EDIT, permissions.RESOURCE_CONNECTION)])
+@provide_session
+def test_connection(connection_id, session):
+    """Test a connection"""
+    try:
+        data = connection_schema.load(request.json, partial=True)
```

Comment:
I’ll suggest that we make this endpoint a GET since what we need is to run test_connection() on a connection instance